multi-tenant/Readme.md

```diff
@@ -59,20 +59,68 @@ docker-compose exec kafka-client kafka-topics --bootstrap-server conduktor-proxy
-echo testMessageParis | docker-compose exec -T kafka-client kafka-console-producer --bootstrap-server conduktor-proxy:6969 --producer.config /clientConfig/paris.properties --topic paris_topic
+echo testMessageLondon | docker-compose exec -T kafka-client kafka-console-producer --bootstrap-server conduktor-proxy:6969 --producer.config /clientConfig/london.properties --topic londonTopic
+echo testMessageParis | docker-compose exec -T kafka-client kafka-console-producer --bootstrap-server conduktor-proxy:6969 --producer.config /clientConfig/paris.properties --topic parisTopic
 ```
 
 ### Step 6: Consume from the topics
 
 And consume the messages produced above. Note that the message `testMessageLondon` will never be consumed by a `Paris` cluster client due to cluster isolation. 
 
 ```bash
-docker-compose exec kafka-client kafka-console-consumer --bootstrap-server conduktor-proxy:6969 --consumer.config /clientConfig/london.properties --topic london_topic --from-beginning
-docker-compose exec kafka-client kafka-console-consumer --bootstrap-server conduktor-proxy:6969 --consumer.config /clientConfig/paris.properties --topic paris_topic --from-beginning
+docker-compose exec kafka-client kafka-console-consumer --bootstrap-server conduktor-proxy:6969 --consumer.config /clientConfig/london.properties --topic londonTopic --from-beginning
+docker-compose exec kafka-client kafka-console-consumer --bootstrap-server conduktor-proxy:6969 --consumer.config /clientConfig/paris.properties --topic parisTopic --from-beginning
 ```
 
-### Step 7: Log into the platform
+### Step 7: Applying Multi Tenancy to existing topics
+
+During migration to Conduktor Proxy you may want to make up a tenant population from existing topics in your Kafka cluster. Conduktor Proxy allows this via administration APIs. In this next section we will create topics on the backing Kafka cluster and add them to tenants within Conduktor Proxy
+
+Let's start by creating some pre-exiting topics and adding data
+
+```bash
+docker-compose exec kafka-client kafka-topics --bootstrap-server kafka1:9092 --create --topic existingLondonTopic
+docker-compose exec kafka-client kafka-topics --bootstrap-server kafka1:9092 --create --topic existingSharedTopic
+echo existingLondonMessage | docker-compose exec -T kafka-client kafka-console-producer --bootstrap-server kafka1:9092 --topic existingLondonTopic
+echo existingSharedMessage | docker-compose exec -T kafka-client kafka-console-producer --bootstrap-server kafka1:9092 --topic existingSharedTopic
+```
+
+### Step 8: Configuring tenants for existing topics
+
+We'll create the following mappings:
+* tenant: `London` can see `existingLondonTopic` and `existingSharedTopic`
+* tenant: `Paris` can see only `existingSharedTopic`
+
+```bash
+docker-compose exec kafka-client curl -X POST conduktor-proxy:8888/topicMappings/1-1/existingLondonTopic -d '{ "topicName":"existingLondonTopic" }'
```

Comment:
should we explain where the 1-1 comes from, and any cases where it might not be 1-1?

Reply:
Actually, that's a bigger request.  Can you explain in the text what the commands are doing, so people understand what they are setting up.  Ie explain what 

topicMappings/1-1/existingLondonTopic -d '{ "topicName":"existingLondonTopic" }'

is setting up.

For tenant 1-1, they are mapping that tenant's view of the topic (in the URL) to a the backing topic (in the -d).

It's not very obvious, because the topic names are the same.

Reply:
we've been trying to downplay this in the demos as it's far from finalised.